dvc/remote/http.py

```diff
@@ -0,0 +1,139 @@
+import os
+import threading
+import requests
+import posixpath
+
+from dvc.logger import Logger
+from dvc.progress import progress
+from dvc.exceptions import DvcException
+from dvc.config import Config
+from dvc.remote.base import RemoteBase
+
+
+class ProgressBarCallback(object):
+    def __init__(self, name, total):
+        self.name = name
+        self.total = total
+        self.current = 0
+        self.lock = threading.Lock()
+
+    def __call__(self, byts):
+        with self.lock:
+            self.current += byts
+            progress.update_target(self.name, self.current, self.total)
+
+
+class RemoteHTTP(RemoteBase):
+    scheme = 'http'
+    REGEX = r'^https?://.*$'
+    REQUEST_TIMEOUT = 10
+    CHUNK_SIZE = 128
```

Comment:
Is there a reason why chunk size is so small?

Reply:
No, I was debugging on small files images with some `sleep`; I think 1MB is better, do you agree?